core/src/banking_stage/qos_service.rs

```diff
@@ -541,109 +476,109 @@ impl QosServiceMetrics {
         if bank_slot != self.slot.load(Ordering::Relaxed) {
             datapoint_info!(
                 "qos-service-stats",
-                ("id", self.id as i64, i64),
```

Comment:
i just cleaned these up while i was in here. unrelated to the actual change - lmk if you'd prefer I separate from the PR